webpack.base.js

```diff
@@ -6,7 +6,7 @@ const { buildEnvKeys, getBuildEnvValue } = require('./scripts/lib/build-env')
 
 const tsconfigPath = path.join(__dirname, 'tsconfig.webpack.json')
 
-module.exports = ({ entry, mode, filename, types, keepBuildEnvVariables }) => ({
+module.exports = ({ entry, mode, filename, types, ignoreWarnings, keepBuildEnvVariables }) => ({
```

Comment:
**💬 suggestion:** ‏with your solution this is not needed, since in `karma.base.conf.js` you don't use `webpackConfig.ignoreWarnings` but define `ignoreWarning` directly instead. You could remove it.